docs/glossary.rst

```diff
@@ -31,41 +48,17 @@ being transferred to an Internet-connected computer.
 Instructions for using SecureDrop as a *Journalist* are available in our
 `Journalist Guide <https://docs.securedrop.org/en/latest/journalist.html>`__.
 
-Application Server
-------------------
-
-The *Application Server* runs the SecureDrop application. This server hosts both
-the website that sources access (the *Source Interface*) and the website that
-journalists access (the *Journalist Interface*). Sources, journalists, and
-admins may only connect to this server using Tor.
-
-Monitor Server
---------------
-
-The *Monitor Server* keeps track of the *Application Server* and sends out an
-email alert if something seems wrong. Only system admins connect
-to this server, and they may only do so using Tor.
-
-Landing Page
-------------
-
-The *Landing Page* is the public-facing webpage for a SecureDrop instance. This
-page is hosted as a standard (i.e. non-Tor) webpage on the news organization's
-site. It provides first instructions for potential sources.
 
-Source Interface
-----------------
-
-The *Source Interface* is the website that sources will access to
-submit documents and communicate with journalists. This site is
-hosted on the *Application Server* and can only be accessed through Tor.
+Journalist Alert Key
+--------------------
+The *Journalist Alert Key* is an alert that notifies journalists daily 
+via encrypted email about whether or not there has been submission activity 
+in the past 24 hours. More information about journalist alerts, is in our 
+`Journalist Guide <https://docs.securedrop.org/en/stable/journalist.html#daily-journalist-alerts-about-submissions>`__.
```

Comment:
We should not use absolute URLs to link to other chapters of the documentation. reStructured text supports the `:doc:` and `:ref:` syntax to cross-reference other chapters and sections, respectively. The glossary chapter has other absolute URL references which should be replaced with `:doc`: or `:ref:`. If you want to take a stab at cleaning up the links in this chapter, please add a commit to do so, otherwise I can do it before merging.